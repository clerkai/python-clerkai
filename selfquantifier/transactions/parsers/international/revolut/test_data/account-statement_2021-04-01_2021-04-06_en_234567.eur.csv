Type,Product,Started Date,Completed Date,Description,Amount,Fee,Currency,State,Balance
TOPUP,Current,2021-04-03 09:50:10,2021-04-03 09:50:12,Apple Pay Top-Up by *1234,100.00,0.00,EUR,COMPLETED,113.00
EXCHANGE,Current,2021-04-03 09:50:29,2021-04-03 09:50:29,Exchanged to SEK,-50.00,0.00,EUR,COMPLETED,63.00
TOPUP,Current,2021-04-06 08:05:15,2021-04-06 08:05:18,Apple Pay Top-Up by *1234,100.00,0.00,EUR,COMPLETED,163.00
EXCHANGE,Current,2021-04-06 08:05:38,2021-04-06 08:05:38,Exchanged to SEK,-48.69,0.00,EUR,COMPLETED,114.31
