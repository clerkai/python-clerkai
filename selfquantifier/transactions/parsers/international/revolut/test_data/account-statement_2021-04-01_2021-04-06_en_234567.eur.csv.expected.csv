Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
2021-04-03 09:50:10,2021-04-03 09:50:12,,Apple Pay Top-Up by *1234,100.00,EUR,113.00,,,,"{'Type': 'TOPUP', 'Product': 'Current', 'Started Date': '2021-04-03 09:50:10', 'Completed Date': '2021-04-03 09:50:12', 'Description': 'Apple Pay Top-Up by *1234', 'Amount': 100.0, 'Fee': 0.0, 'Currency': 'EUR', 'State': 'COMPLETED', 'Balance': 113.0}",2021-04-03 09:50:10,2021-04-03 09:50:12,,Apple Pay Top-Up by *1234,100.0,EUR,113.0,,,
2021-04-03 09:50:29,2021-04-03 09:50:29,,Exchanged to SEK,-50.00,EUR,63.00,,,,"{'Type': 'EXCHANGE', 'Product': 'Current', 'Started Date': '2021-04-03 09:50:29', 'Completed Date': '2021-04-03 09:50:29', 'Description': 'Exchanged to SEK', 'Amount': -50.0, 'Fee': 0.0, 'Currency': 'EUR', 'State': 'COMPLETED', 'Balance': 63.0}",2021-04-03 09:50:29,2021-04-03 09:50:29,,Exchanged to SEK,-50.0,EUR,63.0,,,
2021-04-06 08:05:15,2021-04-06 08:05:18,,Apple Pay Top-Up by *1234,100.00,EUR,163.00,,,,"{'Type': 'TOPUP', 'Product': 'Current', 'Started Date': '2021-04-06 08:05:15', 'Completed Date': '2021-04-06 08:05:18', 'Description': 'Apple Pay Top-Up by *1234', 'Amount': 100.0, 'Fee': 0.0, 'Currency': 'EUR', 'State': 'COMPLETED', 'Balance': 163.0}",2021-04-06 08:05:15,2021-04-06 08:05:18,,Apple Pay Top-Up by *1234,100.0,EUR,163.0,,,
2021-04-06 08:05:38,2021-04-06 08:05:38,,Exchanged to SEK,-48.69,EUR,114.31,,,,"{'Type': 'EXCHANGE', 'Product': 'Current', 'Started Date': '2021-04-06 08:05:38', 'Completed Date': '2021-04-06 08:05:38', 'Description': 'Exchanged to SEK', 'Amount': -48.69, 'Fee': 0.0, 'Currency': 'EUR', 'State': 'COMPLETED', 'Balance': 114.31}",2021-04-06 08:05:38,2021-04-06 08:05:38,,Exchanged to SEK,-48.69,EUR,114.31,,,
